### ISSUE TITLE
Installation should create unixy shortcuts on Windows when invoked through git bash

### ISSUE THREAD

**Issue by jakub-g** [GoogleChromeLabs/jsvu#51]
When installing from git bash on Windows, the tool should create unixy shortcuts instead of `cmd` ones (or actually it could create both, so that things work from `cmd` as well as `bash`).

```
...
Installing wrapper script to ~\.jsvu\spidermonkey.cmd…
...
Installing wrapper script to ~\.jsvu\v8.cmd…
```

git bash can be recognized via `/^MINGW(32|64)$/.test(process.env.MSYSTEM)`

**Comment by mathiasbynens** [comment]
Disclaimer: I don't actively use Windows and know very little about `cmd` or `git bash`.

Do the `*.cmd` ones not work in `git bash`?

**Comment by jakub-g** [comment]
Nope, git bash is bash compiled for Windows, it runs .sh files, it doesn't
run .bat or .cmd files itself directly.

You can invoke a .bat or .cmd via `cmd //c somefile.cmd` but the preferred
way is to use `./somefile.sh` just like on unixy platforms.

I'll try to have a look on it and send a patch.
